mvrx/src/main/kotlin/com/airbnb/mvrx/DelegateMvRxView.kt

```diff
@@ -0,0 +1,39 @@
+package com.airbnb.mvrx
+
+import android.os.Bundle
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.LifecycleEventObserver
+import androidx.lifecycle.LifecycleOwner
+import androidx.savedstate.SavedStateRegistryOwner
+
+class DelegateMvRxView(
+        private val owner: SavedStateRegistryOwner,
```

Comment:
This can be either an `Activity` or `Fragment` Should Google. The type could be more open to allow. for a different method of `mvrxViewId` saving and restoration.

Reply:
this looks pretty reasonable to me. can you add some kdoc to this class to describe the purpose and when it may be used?